src/multi_vector_simulator/E0_evaluation.py

```diff
@@ -136,7 +138,7 @@ def evaluate_dict(dict_values, results_main, results_meta):
                 + dict_values[ENERGY_STORAGE][storage][STORAGE_CAPACITY][
                     OPTIMIZED_ADD_CAP
                 ][UNIT]
-                + ") SOC"
+                + f") {SOC}"
```

Comment:
Just realized that [shorly above this line](https://github.com/rl-institut/multi-vector-simulator/blob/8ad8b71e8461f30286fe7d267432b113615c7156/src/multi_vector_simulator/E0_evaluation.py#L131), the SOC is defined relative only to the `OPTIMIZED_ADD_CAP`. However, I find this a bit weird, as the SOC is relative to `OPTIMIZED_ADD_CAP + INSTALLED_CAP`. Can you change this?

Reply:
:+1: It's just the label that is defined there - I've changed it.